mkrules.cmake

```diff
@@ -105,7 +107,9 @@ function(all_lv_bindings)
 
     file(GLOB_RECURSE LV_PNG_HEADERS ${LV_PNG_DIR}/*.h)
     configure_file(${LV_PNG_DIR}/lodepng.cpp ${LV_PNG_C} COPYONLY)
-    idf_build_set_property(COMPILE_DEFINITIONS "${LV_PNG_PP_OPTIONS}" APPEND)
+    if(ESP_PLATFORM)
+        idf_build_set_property(COMPILE_DEFINITIONS "${LV_PNG_PP_OPTIONS}" APPEND)
```

Comment:
This adds compilation flags for PNG library, which you use on RPi (`LV_PNG`).
So I guess you would need an `else()` which sets it for RPi.

Reply:
Good catch. I used `add_definitions` for non-ESP now.